source/llm/mmri.py

```
@@ -0,0 +1,486 @@
+                    continue
+                _text_elements.append(elem)
+
+            for elem in _text_elements[-3:]:
+                self.driver.execute_script(
+                    "arguments[0].classList.add('ignore_this')", elem
+                )
+                name_elems = elem.find_elements(By.CSS_SELECTOR, ".name")
+                if name_elems:
+                    nickname = name_elems[0].text
+                else:
+                    nickname = "시청자"
```

Ruff (0.12.2)

Use ternary operator `nickname = name_elems[0].text if name_elems else "시청자"` instead of `if`-`else`-block

Replace `if`-`else`-block with `nickname = name_elems[0].text if name_elems else "시청자"`

(SIM108)